authentication/subscription/service/oso_subscription_service.go

```diff
@@ -170,10 +170,50 @@ func (s *osoSubscriptionServiceImpl) loadSubscriptions(ctx context.Context, user
 		log.Error(ctx, map[string]interface{}{
 			"err":         err,
 			"reg_app_url": regAppURL,
+			"username":    username,
 			"body":        bodyString,
 		}, "unable to unmarshal json with subscription status")
 		return nil, autherrors.NewInternalError(ctx, err)
 	}
 
 	return &sbs, nil
 }
+
+// DeactivateUser deactivates the user on OpenShift
+func (s *osoSubscriptionServiceImpl) DeactivateUser(ctx context.Context, username string) error {
+	// Load status from OSO
```

Comment:
Copy-paste from another function. We can just remove that comment.

Reply:
what's wrong with this comment? I can replace "OpenShift" by "OpenShift Online" if you prefer

Reply:
done in d770717

Reply:
You are not loading any status from OSO here. You are deactivating the user. The status is loaded in the other method.

Reply:
To be clear. I'm talking about `// Load status from OSO`. Line 184.

Reply:
ahhh, sorry, I read the wrong line :/ Let me fix that right now

Reply:
fixed in 964c7fd